test/Kestrel.Interop.FunctionalTests/SkipOnJenkinsAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Testing.xunit;
+
+namespace Interop.FunctionalTests
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class, AllowMultiple = false)]
+    public class SkipOnJenkinsAttribute : Attribute, ITestCondition
```

Comment:
This is somewhat temporary until we move to VSTS for check-in validate. We don't have control over the jenkins agents so we can't manage chrome installations easily.

Reply:
Is CI true on jinkins?
Why not combine this with the SkipIfChromeUnavailableAttribute check?

Reply:
Yes CI is true on Jenkins. I didn't want to conflate the two and explain why jenkins is a special case especially since we'll remove this when we move to VSTS. Though I could combine it.